source/imaer-util/src/main/java/nl/overheid/aerius/util/OptionsMetadataUtil.java

```diff
@@ -112,6 +136,20 @@ private static void opsOptionsToMap(final OPSOptions options, final Map<Option,
     }
   }
 
+  private static void ncaOptionsToMap(final Map<Option, String> mapToAddTo, final NCACalculationOptions options, final boolean addDefaults) {
+    if (options != null) {
+      addValue(mapToAddTo, Option.ADMS_PERMIT_AREA, options.getPermitArea(), false);
+      addValue(mapToAddTo, Option.ADMS_METEO_SITE_LOCATION, options.getMeteoSiteLocation(), false);
+      addValue(mapToAddTo, Option.ADMS_METEO_YEARS, String.join(",", options.getMeteoYears()), false);
+      final ADMSOptions adms = options.getAdmsOptions();
+      addValue(mapToAddTo, Option.ADMS_MIN_MONIN_OBUKHOV_LENGTH, adms.getMinMoninObukhovLength(), false);
```

Comment:
Think this can run into nullpointer exceptions? (`NCACalculationOptions ` is not initialized with this object at least)